src/conditionalriskofkmins.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+from scipy.stats import hypergeom
+from scipy.signal import fftconvolve
+import datetime as dt
+
+starttime = dt.datetime.now()
+
+''' This function aids in truncating distributions by finding levels l and u such that 
+    cdf(l) < .0000001 and 1 - cdf(u) < .0000001. (Here, cdf is used somewhat loosely because we do not require cdf(infinity) = 1,
+    although the distribution should sum "close enough" to 1 because .0000001 is absolute, not relative 
+    (i.e. a distribution that summed to .0000004 would result in only half the distribution being between l and u). 
+    
+    The purpose of this is to improve efficiency, since, for instance almost all of the hypergeometric distribution falls between a fraction of its range. 
+    This decreases the time it takes to iterate over the (meaningful parts of) distributions. '''
+def get_interval(dist):
```

Comment:
Lines of code or commentary should not be too long. Long lines are hard to read, review, and force users of some tools like github code review to scroll back and forth.  It can be hard to read highly-indented but short lines also, but I've heard recommendations for maximums of 72, 88, 100, or certainly no more than 110 or so.